src/utils/hooks/send-notification.js

```diff
@@ -26,13 +27,15 @@ const sendNotification = async (message) => {
 
   if (status === FAILED && ['production', 'test'].includes(config.clusterEnv)) {
     try {
-      await sendFailedSlackMessage(message, user, experiment);
+      const stateMachineArn = process === 'qc' ? experiment.meta[OLD_QC_NAME_TO_BE_REMOVED].stateMachineArn : experiment.meta[process].stateMachineArn;
```

Comment:
Can we use the `QC_PROCESS_NAME` constant here?

```suggestion
      const stateMachineArn = process === QC_PROCESS_NAME ? experiment.meta[OLD_QC_NAME_TO_BE_REMOVED].stateMachineArn : experiment.meta[process].stateMachineArn;
```

Reply:
Not necessarily, I just moved some code that was inside `sendFailedSlackMessage` outside because it makes for simpler code in teh api v2 side, I would not mess with this until after v2 is switched

Reply:
I think a different ticket for assessing the what removing `OLD_QC_NAME_TO_BE_REMOVED` and doing it should manage that, not a v2 migration ticket

Reply:
I mean changing `=== 'qc'` to `=== QC_PROCESS_NAME`

Reply:
Ahh ok yes no porblem